query-engine/connector-test-kit-rs/query-engine-tests/tests/new/regressions/prisma_15204.rs

```diff
@@ -38,8 +38,22 @@ mod conversion_error {
         Ok(())
     }
 
-    #[connector_test(schema(schema_bigint))]
-    async fn convert_to_bigint(runner: Runner) -> TestResult<()> {
+    #[connector_test(schema(schema_int), only(Sqlite, JS))]
+    async fn convert_to_int_sqlite_js(runner: Runner) -> TestResult<()> {
+        create_test_data(&runner).await?;
+
+        assert_error!(
+            runner,
+            r#"query { findManyTestModel { field } }"#,
+            2023,
+            "Inconsistent column data: Conversion failed: number must be an integer in column 'field'"
```

Comment:
We can not easily make this error message similar to the Quaint one?

(It's fine, just and error edge case, but double checking as of course less variants would be better)

Reply:
Not exactly the same. No.